components/object-property-list/object-property-list-item.js

```diff
@@ -44,28 +45,33 @@ export class ObjectPropertyListItem extends RtlMixin(LitElement) {
 			:host([dir="rtl"]) .item-icon {
 				margin: -0.1rem 0 0 0.3rem;
 			}
+			:host([skeleton]) .d2l-skeletize {
+				display: inline-flex;
+				height: 0.7rem;
+				width: 100px;
+			}
 		`];
 	}
 
 	render() {
 		return html`
 			${this._renderIcon()}
-			<span>${this.text}</span>
+			<span class="d2l-skeletize">${this.text}</span>
 			${this._renderSeparator()}
 		`;
 	}
 
 	_renderIcon() {
-		return this.icon ? html`<d2l-icon icon="${this.icon}" class="item-icon"></d2l-icon>` : nothing;
+		return this.icon && !this.skeleton ? html`<d2l-icon icon="${this.icon}" class="item-icon"></d2l-icon>` : nothing;
 	}
 
 	_renderSeparator() {
-		return html`
+		return !this.skeleton ? html`
 			<span class="separator">
 				<span class="d2l-offscreen">${getSeparator({ nonBreaking: true })}</span>
 				<d2l-icon icon="tier1:bullet" aria-hidden="true"></d2l-icon>
 			</span>
-		`;
+		` : nothing;
```

Comment:
Is that the case? From the Slack conversation:

> Dave L: Because if we render the separators, then all the spacing is just handled by the existing component
> Jeff G: Oh that would be even better if that’s how it works. I didn’t raise the dot separator because I could take it or leave it, but I like letting the component handle its own spacing instead of approximating it with extra css … nice.

My interpretation of that was that Jeff didn't think we needed the level of detail that the dots provide in the skeleton, but once we mentioned that if we DID show the dots then all the spacing would be handled, he was on board.

Reply:
Just referring to the [doc](http://proddevutil.desire2learn.d2l/d2c/content/product%20design/design.d2l%20archive/Skeleton%20Page%20Loaders%20%E2%80%93%20Daylight%20Design%20System.html) that Louis has shared a few times:

> When multiple elements of dynamic text metadata are concatenated via dots, do not render the dots until the dynamic text is rendered.

Reply:
I'm not sure how much weight we should be putting in a doc from 2020 on the old design.d2l archive at this point... 

@geurts do you feel comfortable weighing in again and overriding Colin's original design? He wanted the dots suppressed, which we can do but not suppressing them gives us the spacing between items for free.

Reply:
Currently we are getting the spacing for free. This is what it looks like without the dots: 
![image](https://user-images.githubusercontent.com/66270369/199280320-6ffa0a01-bed4-419f-be60-c5474e4f158c.png)

Reply:
> Currently we are getting the spacing for free

Actually, that's my old nemesis "arbitrary whitespace" 😂. It's coming from the newlines/tabs between elements, which includes how `d2l-object-property-list-item` is coded internally. If you remove them you lose the space (see first two items in this screenshot):

![image](https://user-images.githubusercontent.com/8449639/199282423-a2c54643-deb4-4952-88c2-b16086ad2b07.png)

I've been [wanting to remove the effect of this whitespace](https://github.com/BrightspaceUI/core/pull/2649), but we haven't come to a solution yet.

Reply:
Back again! 😆  We should come up with a nickname for him... Arby?

Reply:
> I'm not sure how much weight we should be putting in a doc from 2020 on the old design.d2l archive at this point...
> 
> @geurts do you feel comfortable weighing in again and overriding Colin's original design? He wanted the dots suppressed, which we can do but not suppressing them gives us the spacing between items for free.

Yes, we can override that old decision. That whole doc is being reworked for publishing on the Daylight site. 

Can we render the dots in the same colour as the rest of the skeleton, though? If we can't easily do that, then I would leave them out (they will look super weird if they're rendered so much darker than the rest of the skeleton).

Reply:
>Can we render the dots in the same colour as the rest of the skeleton, though?

Yep, we should be able to do that. And yep, when I was playing around with it, the dots did look a little weird unless they were colored the same as the skeleton.